honeybee/radiance/material/glow.py

```diff
@@ -37,6 +37,40 @@ def __init__(self, name, red=0, green=0, blue=0, max_radius=0):
         self.max_radius = max_radius
         """Maximum radius for shadow testing"""
 
+    def to_json(self):
+        """Translate radiance material to json
+        {
+            "type": "glow", // Material type
+            "name": "", // Material Name
+            "red": int, // A positive value for the Red channel of the glow
+            "green": int, // A positive value for the Green channel of the glow
+            "blue": int, // A positive value for the Blue channel of the glow
+            "radius": float // Maximum radius for shadow testing
+        }
+        """
+        return {
+            "type": "glow", 
+            "name": self.name,
+            "red": int(self.red),
```

Comment:
Why input values in glow should be an integer? I believe float is just fine. @AntoineDao do you have a reference for this?